Source/WebCore/css/SelectorPseudoClassAndCompatibilityElementMap.in

```diff
@@ -1,4 +1,5 @@
 -khtml-drag
+-internal-is-html
```

Comment:
Can we rename this and related C++ names to is-html-document ? 

:-internal-is-html can be confusing, since it's unclear if it's the element or the document that is HTML.

Reply:
Just this or also renaming 'matchesIsHtmlPseudoClass' as 'matchesIsHtmlDocumentPseudoClass' etc. Happy to do this since now it works and this is just renaming.

Reply:
Also renaming 'matchesIsHtmlPseudoClass' as 'matchesIsHtmlDocumentPseudoClass'

Reply:
On it. :-)

Reply:
Actually, can you also remove the "Is" part? since the it's not the element that's the html document

Reply:
Same from the pseudo class name: `:-internal-html-document`

Reply:
Ops! Just pushed with 'is' part. Let me do another quick one. :-)

Reply:
[5984003](https://github.com/WebKit/WebKit/commit/59840030d0cd2f621b9445f5a5849878867676a0) now with all asked changes and renames.

Reply:
Commit log also updated. Marking this as resolved.

Reply:
@nt1m - now renamed to 'internal-html-document' and all other related C++ parts as well.